public/robots.txt

```diff
@@ -3,3 +3,5 @@
 # To ban all spiders from the entire site uncomment the next two lines:
 # User-agent: *
 # Disallow: /
+
+Sitemap: https://www.vets.gov/gids/sitemap.xml.gz
```

Comment:
AFAIK robots.txt can only be placed at `/` in a domain. Since GIDS as a whole is deployed to `www.vets.gov/gids`, will this even work?

Reply:
The existing robots.txt is at: https://github.com/department-of-veterans-affairs/vets-website/blob/master/assets/robots.txt and a second `sitemap:` line can be added there to point to this.